test/FlapperUniV2SwapOnly.t.sol

```diff
@@ -0,0 +1,416 @@
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.8.16;
+
+import "dss-test/DssTest.sol";
+import { FlapperUniV2SwapOnly } from "src/FlapperUniV2SwapOnly.sol";
+import "./helpers/UniswapV2Library.sol";
+
+import { DssInstance, MCD } from "dss-test/MCD.sol";
+import { FlapperInstance } from "deploy/FlapperInstance.sol";
+import { FlapperDeploy } from "deploy/FlapperDeploy.sol";
+import { FlapperUniV2Config, FlapperInit } from "deploy/FlapperInit.sol";
+
+interface ChainlogLike {
+    function getAddress(bytes32) external view returns (address);
+}
+
+interface VatLike {
+    function sin(address) external view returns (uint256);
+    function dai(address) external view returns (uint256);
+    function live() external view returns (uint256);
+    function move(address, address, uint256) external;
+    function cage() external;
+}
+
+interface VowLike {
+    function file(bytes32, address) external;
+    function file(bytes32, uint256) external;
+    function rely(address) external;
+    function flap() external returns (uint256);
+    function Sin() external view returns (uint256);
+    function Ash() external view returns (uint256);
+    function heal(uint256) external;
+    function bump() external view returns (uint256);
+    function hump() external view returns (uint256);
+}
+
+interface EndLike {
+    function cage() external;
+}
+
+interface SpotterLike {
+    function par() external view returns (uint256);
+}
+
+interface PairLike {
+    function mint(address) external returns (uint256);
+    function sync() external;
+}
+
+interface GemLike {
+    function balanceOf(address) external view returns (uint256);
+    function approve(address, uint256) external;
+    function transfer(address, uint256) external;
+}
+
+contract MockMedianizer {
+    uint256 public price;
+    mapping (address => uint256) public bud;
+
+    function setPrice(uint256 price_) external {
+        price = price_;
+    }
+
+    function kiss(address a) external {
+        bud[a] = 1;
+    }
+
+    function read() external view returns (bytes32) {
+        require(bud[msg.sender] == 1, "MockMedianizer/not-authorized");
+        return bytes32(price);
+    }
+}
+
+contract FlapperUniV2SwapOnlyTest is DssTest {
+    using stdStorage for StdStorage;
+
+    FlapperUniV2SwapOnly public flapper;
+    FlapperUniV2SwapOnly public linkFlapper;
+    MockMedianizer       public medianizer;
+    MockMedianizer       public linkMedianizer;
+
+    address     DAI_JOIN;
+    address     SPOT;
+    address     DAI;
+    address     MKR;
+    address     USDC;
+    address     LINK;
+    address     PAUSE_PROXY;
+    VatLike     vat;
+    VowLike     vow;
+    EndLike     end;
+    SpotterLike spotter;
+
+    address     constant  LOG           = 0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F;
```

Comment:
```suggestion
    address constant LOG                 = 0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F;
```

Reply:
(And the same nit in test/FlapperUniV2.t.sol)

Reply:
Fixed in both files!